README.md

```diff
@@ -1,11 +1,11 @@
 # libiamf
 
-This is a reference implementation and conformance test for the AOM Immersive Audio Model and Formats (IAMF) specification, approved as a final deliverable by AOM on April 3, 2024.
+This is a reference implementation and conformance test for the AOM Immersive Audio Model and Formats (IAMF) specification, approved as a final deliverable by AOM on _______, 2024.
 The reference implementation includes only the decoder. The description of each folder is as follows:
 
-- The 'code' directory contains the reference decoder for AOM IAMF v1.0.0-errata.
+- The 'code' directory contains the reference decoder for AOM IAMF v1.1.0.
 - The 'proto' and 'tests' directories contain test vectors and test description metadata for conformance tests.
 
 The source code within this repository that implements the normative portions of the IAMF specification is intended to form the IAMF Reference Implementation approved as an AOM Final Deliverable. Any additional functionality within this repository, including functionality that implements non-normative portions of the IAMF specification, is NOT intended to form part of this IAMF Reference Implementation.
 
-The source code has been updated to cover corner cases since the release of v1.0.0-errata.
+The source code has been updated to implement the base-enhanced profile since the release of v1.0.0-errata.
```

Comment:
```suggestion
The source code has been updated to implement the changes including the base-enhanced profile introduced in the v1.1.0 specification since the release of v1.0.0-errata.
```